lib/sql.js

```diff
@@ -1159,8 +1159,9 @@ SQLConnector.prototype._buildWhere = function(model, where) {
  * @returns {string} The ORDER BY clause
  */
 SQLConnector.prototype.buildOrderBy = function(model, order) {
+  var idName = this.idName(model);
```

Comment:
@raymondfeng I agree with that .. each DB may return in differing orders however subsequent calls to the same query should return the same order.  What is odd with this one is that some times the tests pass and sometimes they fail.  The only way I could see that happening is if the inserts are asynchronous and thus inserted in no guaranteed order in which case I think your suggestion is correct.